spec/helpers/application_helper_spec.rb

```diff
@@ -25,7 +25,7 @@ def current_user
     end
 
     it 'returns true if all networks are connected' do
-      3.times { |t| @current_user.services << FactoryGirl.build(:service) }
+      3.times { |t| @current_user.services << FactoryBot.build(:service) }
```

Comment:
- Layout/SpaceInsideBlockBraces: Space between { and | detected.
- Lint/UnusedBlockArgument: Unused block argument - `t`. You can omit the argument if you don't care about it.